src/library/scala/collection/mutable/ArrayDeque.scala

```diff
@@ -83,7 +83,7 @@ class ArrayDeque[A] protected (
     this
   }
 
-  override def prependAll(elems: IterableOnce[A]): this.type = {
+  override def prependAll(elems: IterableOnce[A]) = {
```

Comment:
We should keep the explicit return types (also in other parts of this PR)

Reply:
Isn't the return type redundant (and error-free) when used alongside the override keyword? I am fine either way - I just saw some other places in the code which did not have return type for overrides so I chose to follow one convention. But, I can switch.

Reply:
You're right that the return type is "inherited", but as far as I know we (try to) put the types on public members also in this case.

Reply:
Ok will do!